modules/dr-restoring-cluster-state.adoc

```diff
@@ -97,10 +97,49 @@ The output of this command should be empty. If it is not empty, wait a few minut
 $ sudo mv /var/lib/etcd/ /tmp
 ----
 
+.. If the `/etc/kubernetes/manifests/keepalived.yaml` file exists, follow these steps:
+
+... Move the `/etc/kubernetes/manifests/keepalived.yaml` file out of the kubelet manifest directory:
++
+[source,terminal]
+----
+$ sudo mv /etc/kubernetes/manifests/keepalived.yaml /tmp
+----
+
+... Verify that any containers managed by Keepalived are stopped:
++
+[source,terminal]
+----
+$ sudo crictl ps --name keepalived
+----
++
+The output of this command should be empty. If it is not empty, wait a few minutes and check again.
+
+... Check if the control plane has any Virtual IPs (VIPs) assigned to it:
++
+[source,terminal]
+----
+$ ip -o | egrep '<api_vip>|<ingress_vip>'
+----
+
+... For each reported VIP, run the following command to remove it:
++
+[source,terminal]
+----
+$ sudo ip address del <reported_vip_> dev <reported_vip_device>
+----
+
 .. Repeat this step on each of the other control plane hosts that is not the recovery host.
 
 . Access the recovery control plane host.
 
+. Verify that the recovery control plane node owns the VIP:
++
+[source,terminal]
+----
+$ ip -o | grep <api_vip>
```

Comment:
Same questions regarding `ip -o` here.

Also, does this step only apply if Keepalived was in use? If so, how do we state that for the reader so that they're not running this command unnecessarily?

Are there any troubleshooting steps if this command does not report the expected value?

Reply:
| Does this step only apply if Keepalived was in use?
I believe so, basically it's trying to show that the node has the IP address given to it by the keepalive stuff.

The output **should** look something like this (I think, using the loopback (127.0.0.1) as an example, but it would be an actual ip address in the wild):

> ip -o address | grep -E "127.0.0.1"
1: lo    inet **127.0.0.1**/8 scope host lo\       valid_lft forever preferred_lft forever

If the vip ip is correctly set, then the output will not be empty and the <vip_ip> will be highlighted in the output.
If the vip ip is not set (or not set correctly), then this command will return nothing (empty string).

Reply:
Also, at least on my machine, I didn't have any luck with `ip -o`, I had to do `ip -o address`; could be different.

Reply:
> Also, at least on my machine, I didn't have any luck with ip -o, I had to do ip -o address; could be different.

This is already updated in the PR. :)